### PR TITLE
Association type

### PR DESCRIPTION
**Pull request by bas080**
- leverage aurelia-logging
- fixes for the entity-schema function
- contributing doc replaced

### CODE REVIEW

File: src/entity-schema.js [hunk 1]
```diff
@@ -11,17 +13,29 @@ export function entitySchema(entity) {
...
+      continue;
+    }
+
     let element = {
-      key: key,
+      key : key,
       type: types[key]
     };
 
+    element = extend(true, element, data[key] ? data[key].form || {} : {});
```
Comment (jeremyvergnas) [inline src/entity-schema.js]: What's `.form`?

File: src/aurelia-form.js [hunk 2]
```diff
@@ -76,3 +76,8 @@ export function configure(aurelia, configCallback) {
 }
 
 export {Config};
+
+/* aurelia-form logger */
+import {getLogger} from 'aurelia-logging';
```
Comment (VMBindraban) [inline src/aurelia-form.js]: Move import to top, after that :+1:

**Comment by bas080** [comment]
@jeremyvergnas, It gets my approval for a pull also. :+1:

**Comment by RWOverdijk** [comment]
Not a huge fan of the existy use here, but other than that +1

**Pull request merged**